docs/advanced-usage/compute-backends/kubernetes/index.md

```diff
@@ -108,3 +100,87 @@ You can set `kubernetes_memory_limit` in every `rule` under `resources`.
         "docker://python:2.7-slim"
     ...
 ```
+
+## Job timeout
```

Comment:
`## Custom job timeouts` would play more nicely with "Custom memory limits"

(note also about plurals, may be more intuitive than singulars in titles, since people think usually about a workflow consisting of many jobs of diverse nature)